source/extensions/clusters/aggregate/cluster.cc

```diff
@@ -94,6 +94,7 @@ void Cluster::refresh(const std::function<bool(const std::string&)>& skip_predic
   tls_->runOnAllThreads([this, skip_predicate, cluster_name = this->info()->name()](
                             ThreadLocal::ThreadLocalObjectSharedPtr object)
                             -> ThreadLocal::ThreadLocalObjectSharedPtr {
+    ASSERT(object == nullptr);
```

Comment:
?

Reply:
was checking that this wasn't trying to erase existing data, but now I have the assert in the infrastructure. removed.